package.json

```diff
@@ -151,7 +157,8 @@
     "array.prototype.flatmap": "^1.2.4",
     "cids": "^1.0.0",
     "copy-to-clipboard": "^3.2.0",
-    "d3": "^7.0.0",
+    "d3": "^7.6.1",
```

Comment:
d3 is a big library - have we confirmed that it is being properly tree-shaken?

Reply:
zach is owning looking into this

Reply:
confirmed